src/Opinion/Dto/AverageDto.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Sherl\Sdk\Opinion\Dto;
+
+use JMS\Serializer\Annotation as Serializer;
+
+use Sherl\Sdk\Opinion\Enum\OpinionStatusEnum;
+
+class OpinionUpdateStatusInputDto
+{
+    /**
+     * @var OpinionStatusEnum
+     * @Serializer\Type("Sherl\Sdk\Opinion\Dto\OpinionStatusEnum")
+     */
+    public $status;
+
+    /**
+     * @var string
+     * @Serializer\Type("string")
+     */
+    public $refusedComment;
+}
+
+class AverageDto
```

Comment:
```suggestion
class OpinionAverageDto
```

De manière générale ça me paraissait bien de préfixer les Dto du nom du domaine.